File: C_Programming/Practise_Programs/PP_110.c
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_110
//
//  Description       : This program accepts a string from the user
//                      and displays the string using a separate
//                      function.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 13/08/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(n)
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

void Display(char *str)
{
    printf("Input string is : %s\n", str);
}

int main()
{
    char Arr[50] = {'\0'};

    printf("Enter string : \n");
    scanf("%[^'\n']s", Arr);  
    
    Display(Arr);
    
    return 0;
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      Jay Ganesh
//
//  Sample Output :
//
//      Enter string :
//      Input string is : Jay Ganesh
//
//////////////////////////////////////////////////////////////////

File: Java_Programming/Practise_Programs/PP_110.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_110
//
//  Description       : This program accepts a string from the user
//                      and displays the length of the entered string.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 26/08/2026
//
//  Time Complexity   : O(1)
//  Space Complexity  : O(n)
//
//////////////////////////////////////////////////////////////////

import java.util.Scanner;

class PP_110
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        String Arr = null;

        System.out.println("Enter String : ");
        Arr = sobj.nextLine();

        System.out.println("Length of String is : " + Arr.length());

        sobj.close();
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      Jay Ganesh
//
//  Sample Output :
//
//      Enter String :
//      Length of String is : 10
//
//////////////////////////////////////////////////////////////////